go.mod

```diff
@@ -5,6 +5,7 @@ go 1.15
 require github.com/beego/beego/v2 v2.0.1
 
 require (
+	github.com/ddollar/forego v0.16.1 // indirect
```

Comment:
You added in Makefile `install-dependencies`, and this is not related to the code, so should we omit it?

Reply:
@hoangmirs Removed https://github.com/Lahphim/go-crawler-challenge/pull/33/commits/299435d346c848437cc801c9f272123f7e4941d8